.github/workflows/ci.yml

```diff
@@ -18,32 +18,31 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v4
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@v0.8.1
         with:
-          ref: ${{ github.head_ref }}
-          # needed for 'pre-commit-mirrors-insert-license'
-          fetch-depth: 0
-      - name: Run pre-commit-conda
-        uses: quantco/pre-commit-conda@v1
+          environments: default lint
+      - name: pre-commit
+        run: pixi run pre-commit-run --color=always --show-diff-on-failure
 
   mypy-type-checks:
```

Comment:
When properly checking it in pre-commit (see other comment), we don't need this here as it's already being checked by the pre-commit hook.

Reply:
Removed it!